### DISCUSSION TITLE
Is there a "Raspberry Pi Zero 3" on the roadmap and will it be based on a SoC derivative of Raspberry Pi 4 / Compute Module 4?

### DISCUSSION THREAD

**Discussion by Hedda** [geerlingguy/raspberry-pi-pcie-devices#688] [General]
Are there rumors yet of "Raspberry Pi Zero 3" on the roadmap? Will it be based SoC derivative of Raspberry Pi 5 / Compute Module 5?

For timeline reference:

- Raspberry Pi 4 was first released in 2019, and it has now gone more than 5-years and still no Raspberry Pi Zero derivative  of it yet?

- Raspberry Pi Zero 2 W was first released in 2021 (5-years after Raspberry Pi 3) and uses a SoC derivative of Raspberry Pi 3. 

- Raspberry Pi Zero was first released in 2015 (3-years after Raspberry Pi) and uses a SoC derivative of the original Raspberry Pi.

**Comment by geerlingguy** [comment]
Unfortunately, this repository isn't a good place for speculation, I'd leave that to other forums—especially as the Pi Zero line isn't known to have any PCIe connectivity.
